examples/seismic/tutorials/09_snapshotting.ipynb

```diff
@@ -0,0 +1,806 @@
FYGI630/T/2St9zk4zyfgeQTOvIdSgyg5Rns2ajRLcigOUD4XOBOy7cqRJ591KMdR43atRjkP24NL+b63ds+qIANGsIoYYJVCMVdc92lL4Degk2q+091Id+bW/yv31/Nj0vO0xiw/Ln2J3j7xbDwigWez4rByo+lyoIAGpajcIp4KVpyTqCOYYk9tV/Ak+/A+r39t6+uZYAJ58VTJcTYXpWAbbdaXxWPVr5LNLjKJfKMkHsiXlyLBiS5TBq3iYhN8RE2+8rYOfZsJPgg4NAF+Z2zQCAe2ujtayQRckT2Bjspd4rXjn9zhQx6sq3f55/wvsCi8tpPdbJb5uvOWyYvgdkgwH1usH7/mA8Pa6xNxTsnS+6+dCQ4Tbunzdpv0bRTTQ2e9/MvbZ1JcAs+22WAOduwX8TgALooIqAqAqU15ArMrvnjLl1euYGlFWTjpVTNUCtibRoa3oiM2RibB8P2B4mrB8Knv6FcfkEXP5cIccKWnRgtrVANgY9FdBKmE8pxlJMKbLYyfCQJ0A7VDBxpZA7uBuyWztV1ecw0beH5b9XPlwvSUon0d7Yg248ImjZLQBitb551QXPpnDOz1Z26wK0hbFeCJfDjPmwYZ4rjh/PuMwTXp6PqAdCWwqmlwdMXxaUpzPo+RxaElVNJKM/Rf/veRhqf4YhmVD26O/olU4Rb4VG2rhdg8+d2uk//o7m8IlhDL928zVutV83e/9rm7Gal0mEsExbAKdLLbhsZQCLOas1uUwGuhdWgFYBFjKWk35Z5/7+eHOjlbCgXKgDSWdOZaaRu+1IdqCRcOWiM4DsYFo91HIFoq+eyS+5vxx1hVysD2RoSRrFtdU5nfao1MeeiG1MapXlYzCqOnaPd7h51vT0fSAssv04r2V7vtYY26YGgC/nBacyDcaDMJQIKTU3jUUxj3RhUXBtQBwA6o39kg1gD317ay2Pb5b3btiP48yrmeZy8EQ2Y+skowjgQFAF6m4JjPpOMOp6OaaYf+7f9/4oL2OQzVnncOdClulF38P+rA5+aehHf9B07WHMXPEBPIGjmC7ZGUGwWtDQsndF1Ng+qa5ROZUZEWWzSLMSWUb1RgqJeJXhQ2nLyfuT9+l3LruByVg0dIoKLESPgMLGyhQL+tYkarRWEDOk6l7DtwzM/0R7B9Tv7b19a80FTSPwiTCdCOWkG4LXlK4HU96bCjgtXWCnTejUbkJk9w5q997bLLgWjnsQLdCNIm0OQNqA7JqZvj0qxfli6CDarPivgtnUid8jCMW0JBpPvWpOwcpgX2viOipBf1Z/Xu/zrl9948/avm5gV8/TD4/D4ndKWBYtZ/ZOxgYH9sPhYllk05yQUcIDVAuCMickYE9A95aaswBcYdkpLZlmFh87iNoMEKyaEIwvYoDavNIOponQ7hfUQ8HpxwPWB8LlI+H0o2B7EMhDr89cXwroXFBOhPkzR0yeA+N6kMjyC0CVOI+3zPG+dnwzSmnEEXutWlc0dqDS49Juvpv7Ne5LKCl9slvL4SlIwFsKgEUVFRKNp/WYYwWjClr5Qng5zrg8MORuxeG4gu4ELz9oqaw2M6aXGctDwfxlwvJzsURgxvwQWOyyXtPra2dFdzAkZMPC7hjtdzJKuBHBznV2Q8g9IDziWtNc/qeUq/8tTUgz8LN6h6rJhBzrqAl9JhT7bt3Ue8ncBoxILOqtmQgNzbxiujAEosYUGRVVD7dwqvI1Q0ZvwOjv4GAINK8fuZESZIpu2n9KOoU0ThWkdbe7Z/r2xOTPuye7Gc4wZZo8Zli01q0/tm927qGFdcnWmPwGuEynDkDan2FgU9h+oeN945ruufdr2dfqXWec1xlr7cYT/+3e92pe7Gx8IOt74QY2r7a3udRINOWgPCj1b7SFNzi8/Q6q5BqsIsnKJCv0R0LmRTgQbugSBq4JXS/wW7zav6zzeCMArRvubi4nWxLuL6ANkSuFAOwZbrc7cOOzbOAqGIF50KQRlUFklmBOuEHVkws6rTzyEGysL8stI1MAZAPuPsbOAEDvV+7jlYX4tcGOsU4/Vi1jV2rb5tFkPokZD21usz73Ty79d0D93t7bt9YatAbtSpgfu9C4fK+1auuiG0M5U2RVVMVcrEajCRBLyhMCzIBUthjy1gX/YFVHEpguIMNaa5b/11q6fmxShFBshGVISpatht3beH393wLI/QLp2GQwEOuQJrGRzqEKc2k6hz0b5U7Y77wsselm0Lvrp1uIbz1AppkNACl5dyLplMDKlPXjBpo3+ZwnlgKZMjK5kmfXOlPapG737au0BAazoUFYk9k5DZ94N26k8W5llcjeWs6C5fMGvjSUL6cO6iaGLLPSu//1gPNHwvNfCdsH0ZrtnzYtS1cZ7WkGnxh3f2dML0A5A3yxGOkPpHVmD4LtQ2dl0Er6Tm0K2l3xapPS1ZSepn2PrNfm+Qj6oti97JnLRUYlYFAwqBsdSh87p6DmNnh8M3DPYz0B673VKN+gtGnz7t79XbD8SpheCtaHgu3DjMc/b6BDBd9vaIeK08eCej9heiqYnguOf18wvwiWzxW8tsiyzasaK+ZnMsWLboLi6OcmV3MuE/UyRWUHpM/A8qgZXwfvTAyGGyRtTb0JhobKfLDKqG0yundpKKWhcqcwM3OAptpY6ZmNwO6JziCXNAGQMjpM4RRAwGgMbKugkMXTr2pU5ELDeLvXOuKsOzaPMmWeXX1gaADA1tdorElTqNVDrYCY7Xemdgct27odNa7zqKVzHFDrGPQXwK8BGEXcmQpSdN0708RBRAILOTlW9vrlFvIqhSW7zN2DZ5UVdr4BmHphNNaLX+Jku47Llvw+U/bqE9aqDAXaCjCpAcYP95jYdkPGE/CmgLXkcSqI0J4wovo7bPOVGS5hhDQjnRo2e8iNr19lDHXEHAQCX7dshqDUHzd+Roic6VktJ+ezRgQ0WA4HN8Q70E9yF0CE1g0G853hUP82oJv1JCBCZGIvCCOE5UWQLlPjeDdWTBQ0+
```

Comment:
found in [the Devito documentation](https://www.opesci.org/devito/dimension.html#devito.types.dimension.ConditionalDimension)


ie use the format [text\](hyperlink)